Add theming articles (#997)
telerik/maui-docs: styling-and-themes/customize-the-theme.md

---
title: Customize the Telerik Theme
page_title: Customize the Telerik Theme
description: Learn how to customize the Telerik Theme for your .NET MAUI application and alter the default appearance of the UI for .NET MAUI components.
slug: theming-customization
tags: telerik,.net maui,theme,custom
position: 1
---

## Customize the Telerik Theme

There are several ways to customize the visual appearance of the Telerik .NET MAUI controls depending on the needed changes. You can modify only the colors in a certain color variation and that will affect all the controls or you can make more specific cbange in the styles and templates of a certain Telerik UI for .NET MAUI component.

Either approach requires copying the needed resources in a separate resource dictionary, modifying it and merging it after the Telerik theme resources.

Check the detailed steps below:

### Copy and modify the theme resources

> Inside the **Telerik Theming** folder you have all the theme color variations (inside the **Swatches** folder) as well as the styles and templates of the Telerik .NET MAUI controls (inside the **Styles** folder).

Reviewer: ```suggestion
All available theme styles are in the `TelerikTheming` folder:
* The color variations are in the `Swatches` folder.
* The styles and templates for the Telerik UI for .NET MAUI controls are in the `Styles` folder.
```
